scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -386,6 +386,24 @@ def test_seed_gives_repeatability(self):
         assert_equal(result.x, result2.x)
         assert_equal(result.nfev, result2.nfev)
 
+    def test_random_generator(self):
+        # check that np.random.Generator can be used (numpy >= 1.17)
+        try:
+            # obtain a np.random.Generator object
+            rng = np.random.default_rng()
+        except AttributeError:
```

Comment:
similar comment as above on other test: 
is it not possible to check the numpy version and skip conditionally? 
e.g. using a `@pytest.mark.skipif()` annotator? 
check numpy version is a little different than e.g.
http://doc.pytest.org/en/latest/skipping.html#id1
but similar enough, that it would seem like that would work.

Reply:
this one can be a skipif